sw/device/lib/dif/dif_adc_ctrl.h

```diff
@@ -73,7 +73,7 @@ typedef enum dif_adc_ctrl_channel {
  * instances that may be configured.
  */
 typedef enum dif_adc_ctrl_filter {
-  DIF_ADC_CTRL_FILTER_LIST(DIF_ADC_CTRL_FILTER_ENUM_INIT_)
+  DIF_ADC_CTRL_FILTER_LIST(DIF_ADC_CTRL_FILTER_ENUM_INIT_) kDifAdcCtrlTrans,
```

Comment:
Nit: maybe add a new line between the x-macro instantiation and this additional item in the enum?

Reply:
ok will do!

Reply:
ah that's a clang formatter thing. it only accepts this when leaving a blank line in between (which I think is still better for readability)